plugin/src/main/java/com/iridium/iridiumskyblock/IridiumSkyblock.java

```diff
@@ -361,6 +369,11 @@ public void loadConfigs() {
         this.bankItemList.add(bankItems.moneyBankItem);
 
         this.missionsList = new HashMap<>(missions.missions);
+        this.upgradesList = new HashMap<>(ImmutableMap.<String, Upgrade>builder()
```

Comment:
Why are you using an ImmutableMap? Wouldn't it be nice for developers to have their own upgrades?

Reply:
ye which is why im then wrapping that with new HashMap, theres probabbly a better way to do that but I couldnt think of any

Reply:
Indeed, I didn't see the conversion. I'll look for a shorter way in a few minutes